manifests/init.pp

```diff
@@ -66,6 +67,12 @@
     locale                     => 'en_US.utf8',
   }
 
+  # Starting with PE 2016.1.1; a minimum of 200 max_connections
+  # is recommended
+  postgresql::server::config_entry { 'max_connections':
```

Comment:
In case someone wants to manage this post-install let's allow an option to exclude this resource.